modules/nf-core/bcftools/merge/main.nf

```diff
@@ -8,40 +8,35 @@ process BCFTOOLS_CONVERT {
         'quay.io/biocontainers/bcftools:1.16--hfe4b78e_1' }"
 
     input:
-    tuple val(meta), path(input), path(input_index)
+    tuple val(meta), path(vcfs), path(tbis)
     path bed
     path fasta
+    path fasta_fai
 
     output:
-    tuple val(meta), path("*.vcf.gz"), optional:true , emit: vcf_gz
-    tuple val(meta), path("*.vcf")   , optional:true , emit: vcf
-    tuple val(meta), path("*.bcf.gz"), optional:true , emit: bcf_gz
-    tuple val(meta), path("*.bcf")   , optional:true , emit: bcf
-    path "versions.yml"              , emit: versions
+    tuple val(meta), path("*.{bcf,vcf}{,.gz}"), emit: merged_variants
+    path "versions.yml"                       , emit: versions
```

Comment:
Why is the output file type an option. Why not just use vcf.gz?

Reply:
This is an `nf-core` module which is directly installed into the pipeline. They like to create modules which can work under every circumstance. We only create `vcf.gz` files so only bgzipped VCF files will be in the output